cmd/githubgen/allowlist.txt

```diff
@@ -0,0 +1,28 @@
+Caleb-Hurshman
+Doron-Bargo
+MaxKsyunz
+MitchellGale
+YANG-DB
+ag-ramachandran
+agoallikmaa
+alexvanboxel
+architjugran
+asaharan
+avadhut123pisal
+billmeyer
+eedorenko
+emreyalvac
+keep94
+kiranmayib
+kkujawa-sumo
+leonsp-ai
+liqiangz
+mrsillydog
+oded-dd
+open-telemetry/collector-approvers
```

Comment:
Instead of doing this, why not ignore any usernames with the `open-telemetry/` prefix on it? And fail for any usernames with a `/` on it that don't start with `open-telemetry` to exclude teams from other orgs

Reply:
That's a good move, we can allow any open telemetry team for sure.